server/bin/pbench-server-prep-shim-002.sh

```diff
@@ -187,8 +192,10 @@ while read tbmd5 ;do
         (( nerrs++ ))
         continue
     fi
+
     # mv, as well as cp -a, does not restore the SELinux context properly, so we do it by hand
-    restorecon ${dest}/${tb} ${dest}/${tb}.md5
+    tbname=$(basename ${tb})
```

Comment:
Can't we use the `resultname` variable here instead of creating it again?

Reply:
I tried that at first and then I decided to go with the `tbname` (`resultname` has the `.tar.xz` stripped off, so we need to put it back on here, and I thought that was uglier), but I don't have a strong feeling one way or the other.

Reply:
Changed this and also fixed the log message.